sdk/python/lib/pulumi/automation/_workspace.py

```diff
@@ -59,16 +59,20 @@ class WhoAmIResult:
     user: str
     url: Optional[str]
     organizations: Optional[List[str]]
-
+    tokenInformation: Optional[Mapping[str, Any]]
```

Comment:
I think it would be good if we could replicate the type information we've made available in the other languages (that is, that this is a `TokenInformation` with `name` and `organization`).

Reply:
Done.